package.json

```diff
@@ -15,7 +15,6 @@
     "@react-three/drei": "^9.77.3",
     "@react-three/fiber": "^8.13.3",
```

Comment:
Are we planning to use react-three? I don't see it used in the code.

Reply:
Nah, I used them originally because of animations but it's not needed anymore and they are creating peer conflicts